spec/system/sdg/goals_spec.rb

```diff
@@ -117,5 +117,27 @@
       expect(page).to have_css("div.read-more a", text: "Read more about Life on Land", visible: :hidden)
       expect(page).to have_css("div.read-more a", text: "Read less about Life on Land")
     end
+
+    scenario "has tab target section" do
+      create(:sdg_local_target, code: "15.1.1", title: "SDG local target sample text")
+      visit sdg_goal_path(15)
+
+      within "#target_tabs" do
+        expect(page).to have_content "Targets"
+        expect(page).to have_content "Local targets"
+      end
+
+      within ".tabs-content" do
+        expect(page).to have_content "15.1 By 2020, ensure the conservation, restoration and sustainable use of terrestrial and inland freshwater ecosystems and their services, in particular forests, wetlands, mountains and drylands, in line with obligations under international agreements."
```

Comment:
This proves changing line length limit to 250 will only result in requests to change it to 300 (only way this line wouldn't be an offense) :rofl:.

How about using the first sentence of a different target (this one seems to be long even if we cut the sentence after "ecosystems").